game/game.py

```diff
@@ -1,73 +0,0 @@
-import random
```

Comment:
This game.py file was split up into the 3 files: director.py, cards_class.py, and game_class.py so the root game.py file was not needed anymore